provider/export/tests/test-options.rs

```diff
@@ -193,36 +197,34 @@ fn all_hybrid() {
 fn all_runtime() {
     let exported = export_to_map(
         ExportDriver::new(
-            [LocaleFamily::FULL],
+            [DataLocaleFamily::FULL],
             DeduplicationStrategy::Maximal.into(),
             LocaleFallbacker::new().static_to_owned(),
         ),
         &TestingProvider::with_decimal_symbol_like_data(),
     );
 
     // These are all of the supported locales with deduplication applied.
+    #[rustfmt::skip]
     let locales = [
         "ar",
         // "ar-EG", (same as 'ar')
-        "ar-EG-u-nu-latn", // (same as 'ar-u-nu-latn' but DIFFERENT than 'ar-EG')
-        "ar-u-nu-latn",
+        // "ar-EG/latn", (same as 'ar/latn')
+        "ar/latn",
         "bn",
-        "bn-u-nu-latn",
+        "bn/latn",
         "ccp",
-        "ccp-u-nu-latn",
+        "ccp/latn",
         // "en", (same as 'und')
         // "en-001", (same as 'und')
         // "en-ZA", (same as 'und')
         "es",
-        "es-AR",
-        // "fil", (same as 'und')
-        "fr",
-        // "ja", (same as 'und')
+        "es-AR", // "fil", (same as 'und')
```

Comment:
Thanks, but can you please also pull these other comments that are excluding certain locales back onto their own lines?